docs/user-guide/ze-cics.md

```diff
@@ -0,0 +1,267 @@
+# Using Zowe Explorer CICS Extension
+
+The CICS Extension for Zowe Explorer adds additional functionality to the popular Visual Studio Code extension, [Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe). This extension allows you to interact with CICS regions and programs, and run commands against them.
+
+## Contents
+
+- [Features](#features)
+- [System requirements](#system-requirements)
+- [Installing](#installing)
+    - [Installing from Visual Studio Code](#installing-from-visual-studio-code)
```

Comment:
@samanthasusu Because the title has changed, this link needs an update as well.

Reply:
done